adagucserverEC/CRequest.cpp

```diff
@@ -1823,7 +1823,56 @@ int CRequest::process_all_layers() {
       // dataSources[j]->getCDFDims()->addDimension("none","0",0);
     }
     if (dataSources[j]->dLayerType == CConfigReaderLayerTypeLiveUpdate) {
+      // This layer has no dimensions, but we need to add one timestep with data in order to make the next code work.
       layerTypeLiveUpdateConfigureDimensionsInDataSource(dataSources[j]);
+      if (dataSources[j]->cfgLayer->DataPostProc.empty()) {
+        // // Add step to empty file
+        dataSources[j]->addStep("", NULL);
+        dataSources[j]->getCDFDims()->addDimension("none", "0", 0);
+      } else {
+        if (dataSources[j]->cfgLayer->Dimension.size() != 0) {
```

Comment:
Is this for the solar terminator?

Reply:
Yes, indeed. I think I will add some comments to specify exactly why this code is here.